### PR TITLE
[applepayjs] Add billingContact to ApplePayPaymentMethod

### PR DESCRIPTION
**Pull request by vsashyn**
Resolves: https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/57506
[Apple Pay docs]: https://developer.apple.com/documentation/apple_pay_on_the_web/applepaypaymentmethod

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://developer.apple.com/documentation/apple_pay_on_the_web/applepaypaymentmethod>>

### CODE REVIEW

File: types/applepayjs/tslint.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "extends": "@definitelytyped/dtslint/dt.json",
```
Comment (sandersn) [inline types/applepayjs/tslint.json:2]: this changed recently; it needs to be `@definitelytyped/dtslint/dt.json`

CI was broken and should pass once it's re-run with this change reverted.

File: types/applepayjs/index.d.ts [hunk 2]
```diff
@@ -455,6 +455,11 @@ declare namespace ApplePayJS {
          * The payment pass object currently selected to complete the payment.
          */
         paymentPass: ApplePayPaymentPass;
+
+        /**
+         * The billing contact selected by the user for this transaction.
```
Comment (martincostello) [inline types/applepayjs/index.d.ts:460]: I'd suggest matching the other members and using the exact text from the Apple documentation.

```suggestion
         * The billing contact associated with the card.
```

Comment (vsashyn) [reply types/applepayjs/index.d.ts]: Definitely, thanks!

**Review by martincostello — APPROVED** [review]

**Comment by vsashyn** [comment]
Ready to merge

**Pull request merged**